src/rust/zh.manhuagui/res/settings.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "type": "group",
+        "title": "Rate Limit",
+        "items": [
+            {
+                "type": "stepper",
+                "key": "rateLimit",
+                "title": "Number of requests",
+                "default": "5",
```

Comment:
the [documentation](https://gist.github.com/beerpiss/18e0a861a55f6fa9ed6733798f027ee0#stepper) said the type of `default` should be `string`, but it could not pass the `aidoku verify`

Reply:
oh that should be a float https://github.com/Aidoku/Aidoku/blob/8354142a8212b86096010adec5e8ed1bb6259c25/Shared/Sources/Source.swift#L223